### ISSUE TITLE
False positives of missing translations

### ISSUE THREAD

**Issue by samdark** [kalessil/yii2inspections#10]
There are false positives of "The message doesn't have any translations or doesn't belong to the category".

Translations are in `messages/ru-RU`. Config is the following:

```php
return [
    'translations' => [
        '*' => [
            'class' => 'yii\i18n\PhpMessageSource',
            'fileMap' => [
                'user' => 'user.php',
                'stash' => 'stash.php',
                'site' => 'site.php',
            ],
        ],
    ],
];
```

**Comment by kalessil** [comment]
This was a bug from me, now should work (binary pushed)
